ui/page/dexclient/markets.go

```diff
@@ -0,0 +1,137 @@
+package dexclient
+
+import (
+	"context"
+	"fmt"
+
+	"decred.org/dcrdex/client/core"
+	"gioui.org/layout"
+
+	"github.com/planetdecred/godcr/ui/decredmaterial"
+	"github.com/planetdecred/godcr/ui/load"
+	"github.com/planetdecred/godcr/ui/values"
+)
+
+type (
+	C = layout.Context
+	D = layout.Dimensions
+)
+
+const MarketPageID = "Markets"
+
+type Page struct {
+	*load.Load
+	ctx             context.Context // page context
+	ctxCancel       context.CancelFunc
+	initializeModal bool
+	addBTCWallet    decredmaterial.Button
+	advancedTrade   decredmaterial.Button
+}
+
+// TODO: Add collapsible button to select a market.
+// Use mktName="DCR-BTC" in the meantime.
+func (pg *Page) dexMarket(mktName string) *core.Market {
+	dex := pg.dex()
+	if dex == nil {
+		return nil
+	}
+	return dex.Markets[mktName]
+}
+
+func NewMarketPage(l *load.Load) *Page {
+	pg := &Page{
+		Load:            l,
+		initializeModal: false,
+	}
+
+	return pg
+}
+
+func (pg *Page) ID() string {
+	return MarketPageID
+}
+
+func (pg *Page) OnResume() {
+	pg.ctx, pg.ctxCancel = context.WithCancel(context.TODO())
+
+	if pg.Dexc.Core.User().Initialized && pg.Dexc.IsLoggedIn {
+		go pg.readNotifications()
+	}
```

Comment:
The if statement surrounding `go pg.readNotifications()` is unnecessary and prevents catching fee payment notifications after the user registers. The result is that the page continues to display `Waiting for confirmations` even after the fee is confirmed because the notification listener is not enabled. It's safe to just call `go pg.readNotifications()` directly, without an if surrounding it.

Reply:
right!